pajbot/modules/moderators_refresh.py

```diff
@@ -63,12 +62,8 @@ def _update_moderators(self):
             else:
                 log.error(f"Failed to update moderators: {e} - {e.response.text}")
 
-    @time_method
-    def _process_moderator_ids(self, moderator_ids):
-        moderator_basics = self.bot.twitch_helix_api.bulk_get_user_basics_by_id(moderator_ids)
-
-        # filter out invalid/deleted/etc. users
-        moderator_basics = [e for e in moderator_basics if e is not None]
+        # filter out deleted/invalid users
+        moderator_basics = [e for e in moderators if e is not None]
```

Comment:
Won't be necessary anymore, there are not going to be any `None` values in the list

```suggestion
```

You'll just have to rename usages of `moderator_basics` below to instead just use `moderators`.